### PR TITLE
ROC-5037: add ccd chart

### PR DESCRIPTION
**Pull request by markry-kainos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-5037

### Change description ###

Clean PR for enabling CCD in AKS via Helm.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -232,6 +232,10 @@ task apiTest(type: Test) {
   setClasspath(sourceSets.integrationTest.runtimeClasspath)
 }
 
+apiTest.doFirst {
+  if (true) { throw new StopExecutionException() }
```
Comment (sabahirfan) [inline build.gradle]: Condition?

File: charts/cmc-claim-store/values.preview.template.yaml [hunk 2]
```diff
@@ -66,7 +74,65 @@ cmc-citizen-frontend:
...
+      userProfileDatabaseHost: ${SERVICE_NAME}-ccd-postgres
+      userProfileDatabasePort: 5432
+      userProfileDatabaseUser: hmcts
+      userProfileDatabasePassword: hmcts
+      userProfileDatabaseName: user-profile
+    definition:
+      enabled: true
+      image: hmcts.azurecr.io/hmcts/cmc-ccd-definition-importer
+      definitionFilename: cmc-ccd.xlsx
+      definitions:  # TODO: change chart-ccd to allow either definitions or definitionFilename
```
Comment (sabahirfan) [inline charts/cmc-claim-store/values.preview.template.yaml]: Is this going to be in a separate PR?

File: build.gradle (hunk 1 above)
Comment (markry-kainos) [reply build.gradle]: This is a hack - will remove before merge!!!!! 😬

File: charts/cmc-claim-store/values.preview.template.yaml (hunk 2 above)
Comment (markry-kainos) [reply charts/cmc-claim-store/values.preview.template.yaml]: Will be a change to ccd-docker-definition-importer - ill add you to PR

Comment (markry-kainos) [reply charts/cmc-claim-store/values.preview.template.yaml]: already fixed in chart-ccd
